src/components/Form/Form.js

```diff
@@ -0,0 +1,31 @@
+import React, { useState } from 'react'
+import styled from 'styled-components'
+import PropTypes from 'prop-types'
+
+export default function Form({ habits, onSubmit }) {
+  return (
+    <>
+      <FormContainer onSubmit={handleSubmit}>
+        <InputStyled name="habit" type="text" placeholder="Insert a habit" />
+      </FormContainer>
+    </>
+  )
+
+  function handleSubmit(e) {
+    e.preventDefault()
+    const form = e.target
+    const input = form.habit
+    onSubmit(input.value)
+    form.reset()
+  }
+}
+
+const FormContainer = styled.form`
+  margin-top: 20px;
+  display: flex;
+  justify-content: center;
+`
+const InputStyled = styled.input`
+  width: 300px;
```

Comment:
Maybe think of relative values `rem` to provide a better experience on different screen sizes.